chain/client/src/gc_actor.rs

```diff
@@ -0,0 +1,102 @@
+use crate::metrics;
+use actix::{Actor, Addr, Arbiter, ArbiterHandle, AsyncContext, Context};
+use near_chain::{types::RuntimeAdapter, ChainStore, ChainStoreAccess};
+use near_chain_configs::{ClientConfig, GCConfig};
+use near_epoch_manager::EpochManagerAdapter;
+use near_primitives::types::BlockHeight;
+use near_store::{metadata::DbKind, Store};
+use std::sync::Arc;
+use tracing::warn;
+
+pub struct GCActor {
```

Comment:
nit: Add a comment please. 

random remark: I think it makes sense to have gc in a dedicated actor instead of adding it to e.g. to the SyncJobsActor to make sure it's not blocked by any lengthy job that may be sitting there (loading trie, resharding..).